<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTime</class>
 <widget class="QDialog" name="addTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="time_Label">
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Add Time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="name_LineEdit">
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="start_Label">
     <property name="text">
      <string>Start Time:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTimeEdit" name="start_timeEdit">
     <property name="displayFormat">
      <string>h:mm:ss AP</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="end_Label">
     <property name="text">
      <string>End Time:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTimeEdit" name="end_timeEdit">
     <property name="displayFormat">
      <string>h:mm:ss AP</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="volume_Label">
     <property name="text">
      <string>Volume:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QSlider" name="volume_Slider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="fade_checkBox">
     <property name="text">
      <string>Fade In and Out</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addTime</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addTime</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
